fetch.sh

```diff
@@ -39,7 +45,10 @@ QCOW2="$(basename "$TMP_DIR"/*.ovf .ovf).qcow2"
 qemu-img convert -f vmdk -O qcow2 "$VMDK" "$QCOW2"
 
 # Remove now-useless files
-rm "$VMDK" "$TMP_DIR"/*.ovf
+rm "$VMDK" "$TMP_DIR"/*.ov[fa]
 
-echo Finished! Delete "$TMP_DIR" to tidy up
+echo Finished! Deleting "$TMP_DIR" to tidy up
+ls -la "$TMP_DIR"
+rm -fr "$TMP_DIR"
+ln -sf "$QCOW2" "disk.qcow2"
```

Comment:
Is this to have a "default" image? Most people using this I'm guessing will be downloading multiple versions, so this symlink will just fight between whatever you downloaded.

What we could do though is make start.sh search for the most recent qcow2 image if none is given, which would be slightly neater.